src/components/DuosFooter.js

```diff
@@ -1,32 +1,23 @@
-import { Component } from 'react';
 import { a, div, footer, img, li, ul } from 'react-hyperscript-helpers';
 import { Storage } from '../libs/storage';
 
+const DuosFooter = (props) => {
+  const isAdmin = props.isLogged ? Storage.getCurrentUser().isAdmin : false;
+  const supportLink = isAdmin ? '/help_reports' : '/home_help';
 
-class DuosFooter extends Component {
-
-  render() {
-    let isAdmin = false;
-    if (this.props.isLogged) {
-      isAdmin = Storage.getCurrentUser().isAdmin;
-    }
-    const  supportLink = isAdmin ? '/help_reports' : '/home_help';
-
-    return (
-      div({ className: "footer" }, [
-        footer({ className: "main-footer" }, [
-          ul({ className: "footer-links" }, [
-            li({ className: "footer-links__item" }, ["\u00A9 Broad Institute"]),
-            li({ className: "footer-links__item" }, [a({ target: '_blank', href: "https://www.broadinstitute.org/privacy-policy" }, ["Privacy Policy"]),]),
-            li({ className: "footer-links__item" }, [a({ target: '_blank', href: "https://www.broadinstitute.org/terms-conditions" }, ["Terms of Service"]),]),
-            li({ className: "footer-links__item" }, [a({ href: supportLink }, ["Support"]),]),
-            li({ className: 'footer-links__item' }, [a({ href: '/status' }, ['Status'])])
-          ]),
-          img({ src: "/images/broad_logo.svg", className: "footer-logo", alt: "Broad Institute logo" })
+  return (
+    div({ className: "footer" }, [
+      footer({ className: "main-footer" }, [
+        img({ src: "/images/broad_logo_allwhite.png", className: "footer-logo", alt: "Broad Institute logo" }),
+        ul({ className: "footer-links" }, [
+          li({ className: "footer-links__item" }, ["\u00A9 Broad Institute"]),
+          li({ className: "footer-links__item" }, [a({ target: '_blank', href: "https://www.broadinstitute.org/privacy-policy" }, ["Privacy Policy"]),]),
+          li({ className: "footer-links__item" }, [a({ target: '_blank', href: "https://www.broadinstitute.org/terms-conditions" }, ["Terms of Service"]),]),
+          li({ className: "footer-links__item" }, [a({ href: supportLink }, ["Support"]),])
```

Comment:
Pretty sure the test is failing because `li({ className: 'footer-links__item' }, [a({ href: '/status' }, ['Status'])])` is missing.

Reply:
To test the very few integration tests we do have, do this.
* Terminal window 1: `npm start`
* Terminal window 2: `npm run-script cypress:run`